File: Javascript/ContadorInteiro.js
function contarValoresInteiros(primeiroDado, N) {
  let contador = 0;

  for (let i = primeiroDado; i <= N; i++) {
    contador++;
  }

  return contador;
}

const primeiroDado = 5;
const N = 20;

const quantidadeValoresInteiros = contarValoresInteiros(primeiroDado, N);

console.log(
  `Existem ${quantidadeValoresInteiros} valores inteiros entre ${primeiroDado} e ${N}.`
);

File: Java/ContadorInteiro.java
public class ContadorInteiro {
    public static void main(String[] args) {
        int primeiroDado = 5;
        int N = 20;

        int quantidadeValoresInteiros = contarValoresInteiros(primeiroDado, N);

        System.out.println(
                "Existem " + quantidadeValoresInteiros + " valores inteiros entre " + primeiroDado + " e " + N + ".");
    }

    public static int contarValoresInteiros(int primeiroDado, int N) {
        int contador = 0;

        for (int i = primeiroDado; i <= N; i++) {
            contador++;
        }

        return contador;
    }
}
